python/paddle/v2/framework/tests/test_image_classification_train.py

```diff
@@ -75,8 +189,11 @@ def conv_block(input,
     data_type='int64',
     program=program,
     init_program=init_program)
-vgg_net = vgg16_bn_drop(images, program, init_program)
-predict = layers.fc(input=vgg_net,
+# net = vgg16_bn_drop(images, program, init_program)
```

Comment:
Do we need a comment to hint the users switch between VGG and ResNet?

Reply:
done